packages/web/src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Box, HStack, Spacer } from '@chakra-ui/react';
+import { NavLink } from './NavLink';
+import { NavProfileMenu } from './NavProfileMenu';
+
+export const NavBar: React.FC = () => {
+  return (
+    <HStack className="navbar">
```

Comment:
Instead of the class we can just style it here. (We don't need a variant because it's only ever being used here)

```suggestion
export const NavBar: React.FC = () => {
  const navBackgroundColor = useColorModeValue('gray.100', 'gray.900');

  return (
    <HStack backgroundColor={navBackgroundColor}>
```

Reply:
done